server/routes/changeLocation/handlers/changeLocation.ts

```diff
@@ -11,15 +11,17 @@ export default class ChangeLocationRoutes {
       const prisonCaseloadFromNomis = await this.userService.getPrisonUserCaseloads(user)
       const caseload = prisonCaseloadFromNomis.map(c => ({ value: c.caseLoadId, text: c.description }))
       const checked = req.session.caseloadsSelected
-      const cancelLink = role === AuthRole.CASE_ADMIN ? '/licence/view/cases' : '/licence/approve/cases'
+      const cancelToUrl = req.query.view ? '/licence/view/cases?view=probation' : '/licence/view/cases'
```

Comment:
Non-blocking, but I'm not sure if I agree with the approach of having a generic `view` param on the query and only checking if it's defined before setting the view to `probation`.
I understand that `prison` is the default, so if a view is defined then it is currently safe to assume that it will only be `probation`, but I think having `req.query.view === 'probation' ? ...` would make the code easier to follow (without the assumption of prison being the default) and also would allow us to expand the `view` param more easily in future.

Reply:
good point.